ui/helpers/utils/util.js

```diff
@@ -102,7 +103,7 @@ export function addressSummary(
   if (!address) {
     return '';
   }
-  let checked = toChecksumHexAddress(address);
+  let checked = normalizeSafeAddress(address);
   if (!includeHex) {
```

Comment:
I wonder if we should change the behavior of this function, like we honor the `includeHex` if the address (`checked`) starts with `0x`, so that the caller do not have to take care of this in a multichain context, WDYT?

Also, we could have a new "multichain" variant of this one, maybe in a new "multichain utils" module (so we could keep the same function name, but this might get confusing)

Reply:
I think we can leave it as is for now. I would prefer to use `normalizeSafeAddress` everywhere